acra/configuring-maintaining/general-configuration/acra_translator.md

```diff
@@ -0,0 +1,312 @@
+
+* `--jaeger_basic_auth_username=<username>`
+
+  Username used for basic auth (optional) to jaeger.
+
+* `--jaeger_collector_endpoint=<url>`
+
+  Jaeger endpoint that will be used to export trace data.
+  Example: `http://localhost:14268/api/traces`.
+  Default is empty.
+
+* `--tracing_jaeger_enable={true|false}`
+
+  Export trace data to jaeger.
+  Default is `false`.
+
+### Logging
+
+* `--log_to_console={true|false}`
+
+  Log to stderr.
+  Default is `true`.
+
+* `--log_to_file=<filename>`
+
+  Log to file if non-empty value was passed.
+  Default is empty.
+
+* `--logging_format={plaintext|json|CEF}`
+
+  Logging format.
+
+  * `plaintext` — (default) pretty human readable key/value format  
+    `time="2021-07-12T14:02:12+03:00" level=info msg="Starting service acra-translator [pid=475995]" version=0.85.0`
+
+  * `json` — one JSON object per line, easy to parse by most log collectors  
+    `{"level":"info","msg":"Starting service acra-translator [pid=476077]","product":"acra-translator","timestamp":"2021-07-12T14:02:50+03:00","unixTime":"1626087770.004","version":"0.85.0"}`
+
+  * `CEF` — Common Event Format  
+    `CEF:0|cossacklabs|acra-translator|0.85.0|100|Starting service acra-translator [pid\=476133]|1|unixTime=1626087782.510`
+
+* `--tracing_log_enable={true|false}`
+
+  Export trace data to log.
+  Default is `false`.
+
+* `-v`
+
+  Log to stderr all `INFO`, `WARNING` and `ERROR` logs.
+
+* `-d`
+
+  Log everything to stderr.
+
+### Vault
+
+* `--vault_connection_api_string=<url>`
+
+  Connection string (like `http://x.x.x.x:yyyy`) for loading `ACRA_MASTER_KEY` from HashiCorp Vault.
+  Default is empty (`ACRA_MASTER_KEY` environment variable is expected).
+
+* `--vault_secrets_path=<path>`
+
+  KV Secret Path for reading `ACRA_MASTER_KEY` from HashiCorp Vault.
+  Default is `secret/`.
+
+* `--vault_tls_ca_path=<filename>`
+
+  Path to CA certificate for HashiCorp Vault certificate validation.
+  Default is empty (use root certificates configured in system).
+
+* `--vault_tls_client_cert=<filename>`
+
+  Path to client TLS certificate for reading `ACRA_MASTER_KEY` from HashiCorp Vault.
+  Default is empty (don't send client certificate).
+
+* `--vault_tls_client_key=<filename>`
+
+  Path to private key of the client TLS certificate for reading `ACRA_MASTER_KEY` from HashiCorp Vault.
+  Default is empty (don't send client certificate).
+
+* `--vault_tls_transport_enable={true|false}`
+
+  Use TLS to encrypt transport with HashiCorp Vault.
+  Default is `false`.
+
+## Keys
+
+There are few different kinds of keys for different purposes.
+When generating them, few things should be kept in mind:
+* `ACRA_MASTER_KEY` should be set containing master key of same keystore version
+* Output directory should have no permissions for group and others
+
+`client1` in examples is the identifier of client that can be either sent in
+RPC requests or derived from the client TLS certificate depending on configuration.
```

Comment:
IMO, this should go on some other page, not mixed with listing of command-line options.